packages/larva-patterns/utils/getSocialPlatforms.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Get social platform information
+ *
+ * @param {Array} platforms "name" property values of desired platforms (see ../_data/social-platforms.json).
+ */
+module.exports = function( platforms ) {
```

Comment:
I dig this solution. Maybe if anything, it could use more documentation to clarify our various use cases. For example, why it's used like this sometimes: 

```
const socialPlatforms = require( '../../utils/getSocialPlatforms' )( [
	'facebook', 'twitter', 'pinit', 'tumblr', 'reddit', 'linkedin', 'print'
] );
```

and like this other times:

```
const socialPlatforms = require( '../../utils/getSocialPlatforms' )();
```

Basically, clarifying what happens when calling the function without a `platforms` param -- just to make this function easier to understand at a glance.